lib/createStylelintResult.js

```diff
@@ -58,7 +58,7 @@ const _ = require("lodash");
 }
 */
 
-/*:: type resultType = {
+/*:: type configForFileT = {
```

Comment:
@CAYdenberg In my ongoing attempt at learning and understanding Flow better, why the name change here? Also should it be `configForFileType` or are these not really about making them _human-readable_?

Reply:
I found it confusing when reading originally that this type is declared in a file called createStylelintResult, but this is NOT the type of the result. That is stylelint$result (returned on line 79). 

So I looked for another name ... this is the type of the returned value of `getConfigForFile` (encased in a promise), so `configForFile` is what it is. Possibly the variable name should have been changed too.

I believe the capital T is used in other places but not here, so I'll change that too.

Reply:
Sweet, thanks for the explanation, and the follow up commit 👍